internal/server/iigointernal/judiciary_test.go

```diff
@@ -1977,8 +1977,10 @@ func defaultInitJudiciary() judiciary {
 		sanctionRecord:        map[shared.ClientID]shared.IIGOSanctionsScore{},
 		sanctionThresholds:    map[shared.IIGOSanctionsTier]shared.IIGOSanctionsScore{},
 		ruleViolationSeverity: map[string]shared.IIGOSanctionsScore{},
-		gameConf:              &config.IIGOConfig{},
-		gameState:             &gamestate,
+		gameConf: &config.IIGOConfig{
+			DefaultSanctionScore: 5,
+		},
```

Comment:
![image](https://user-images.githubusercontent.com/33488131/104108758-251fbc00-52bf-11eb-8d51-8ffb96387527.png)

Reply:
How do you have these memes ready to go XD

Reply:
Yeah it's just for a test so it'll always use that value to test that function.

Reply:
at fb, we had macros to easily get memes in. I miss that so much. I just google now.